frontend/public/kubevirt/components/vm/vm-detail.jsx

```diff
@@ -30,31 +32,51 @@ import { Disk } from '../disk';
 import { NodeLink, LoadingInline } from '../../../components/utils';
 import { menuActions } from './menu-actions';
 
-const VmiEvents = ({obj: vm}) => {
+const VmEvents = ({ obj: vm }) => {
   const vmi = {
     kind: VirtualMachineInstanceModel.kind,
     metadata: {
-      name: vm.metadata.name,
-      namespace: vm.metadata.namespace,
+      name: getName(vm),
+      namespace: getNamespace(vm),
     },
   };
-  return <ResourceEventStream obj={vmi} />;
+  const launcherPod = {
+    customFilter: ({ kind, namespace, name }) =>
+      kind === PodModel.kind && namespace === getNamespace(vm) && name.startsWith(`${VIRT_LAUNCHER_POD_PREFIX}${getName(vm)}-`),
```

Comment:
note: I think it would be also better to remove the `VIRT_LAUNCHER_POD_PREFIX` in the long run.